code_challenges/code_wars/who_likes_it.py

```diff
@@ -0,0 +1,22 @@
+# https://www.codewars.com/kata/5266876b8f4bf2da9b000362
+
+
+def who_likes(names):
+    if len(names) == 0:
+        names = "no one likes this"
+    elif len(names) == 1:
+        names = names[0] + " likes this"
+    elif len(names) == 2:
+        names = names[0] + " and " + names[1] + " like this"
+    elif len(names) == 3:
+        names = names[0] + ", " + names[1] + " and " + names[2] + " like this"
+    elif len(names) > 3:
+        names = (
+            names[0]
+            + ", "
+            + names[1]
+            + " and "
+            + str(len(names) - 2)
+            + " others like this"
+        )
```

Comment:
It's fine to use concatenation like this, but I find f-string interpolation a bit easier to read

```python
elif len(names) == 1:
    names = names[0] + " likes this"
elif len(names) == 2:
    names = f"{names[0]} and {names[1]} like this"
elif len(names) == 3:
    names = f"{names[0]}, {names[1]} and {names[2]} like this"
elif len(names) > 3:
    names = f"{names[0]}, {names[1]} and {str(len(names) - 2)} others like this"
```

Reply:
I do too!! I think I must've tried using it like that in the past and messed it up so I just have been under the impression f-string could only be used with the print function.